.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,23 @@
+name: E2E tests
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      E2E_TESTS: true
+      E2E_LOGS: true
+      CI_VERBOSE: true
```

Comment:
Currently we aren't utilizing anywhere in the edge codebase these environment variables. I don't mind leaving them if it doesn't break anything, but just saying. :slightly_smiling_face:

Reply:
Good question, also took this into account, considering that we will be moving all e2e from polybft here too, I considered to leave them here in advance, WDYT?

Reply:
Yes sure. I would also consider setting log level to debug (this is something I found useful multiple times on v3, although we are running there on `info` log level, which on some occasions wasn't enough).

Double check, but think this is the correct way to set it:
`E2E_LOG_LEVEL: debug`

Reply:
Done in the next PR #771